packages/components/molecules/f-searchbox/src/components/formElements/FormFullAddressSearchSuggestions.vue

```diff
@@ -62,7 +94,32 @@ export default {
         }
     },
 
+    watch: {
+        /**
+         * Checks the `keyboardSuggestionIndex` value so we can scroll to the address within the
+         * `fullAddressSuggestions` container, allowing keyboard navigation.
+         *
+         * @param {Number} value
+         */
+        keyboardSuggestionIndex (value) {
+            if ((value || value === 0)
+                && this.$refs[`suggestion${[value]}`]
```

Comment:
Might be good to wrap `this.$refs[`suggestion${[value]}`]` into a variable so that you only have to get the value one time?

Reply:
Good shout, fixed.

Reply:
@kevinrodrigues Did you fix this? I can still see `this.$refs[`suggestion${[value]}` here?